app/src/scenes/layouts/data.ts

```diff
@@ -88,6 +90,16 @@ export const data: LayoutData[] = [
       );
     },
   },
+  {
+    title: '相机',
```

Comment:
I plan to put the camera icon to each 'catalog page', i.e., when you click 'Fish' button, you'll see a list of fish icons, and a camera button at the top right corner.
The benefit of this is that we can have one `traineddata` file per category, so that DIY icon won't be recognized as a fish. The a high precision rate could be easily achieved.

I'll make this change after merging your PR.